python/dgl/data/__init__.py

```diff
@@ -70,6 +70,12 @@
     RomanEmpireDataset,
     TolokersDataset,
 )
+
+# RDKit is required for OGB-LSC PCQM4Mv2 and datasets derived from it.
```

Comment:
```suggestion
# RDKit is required for OGB-LSC PCQM4Mv2 and the datasets derived from it.
# Exception handling was added to prevent crashes for users who are using other
# datasets.
```